frontend/src/components/mine/Summary/MineSummary.js

```diff
@@ -190,7 +196,9 @@ export const MineSummary = (props) => {
                       <img alt="Open Orders" src={DOC} style={{ height: 40, paddingRight: 5 }} />
                       &nbsp;
                       <span className="info-display">
-                        {props.mineComplianceInfo.num_open_orders}
+                        {props.complianceInfoLoading
+                          ? String.LOADING
+                          : props.mineComplianceInfo.num_open_orders}
```

Comment:
Do we need any safety checks on `props.mineComplianceInfo` to avoid `Cannot read property num_open_orders of undefined`?

Reply:
This entire section is wrapped in props.mineComplianceInfo && so only runs if mineComplianceInfo exists.